src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,38 @@
+package pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends AbstractPage {
+
+    private final By emailLocator = By.cssSelector("input#email");
```

Comment:
'#email' is already unique. It's better not to add unneeded details to the selector. it makes locators more fragile. For 'input' it's not so relevant, but for example 'div#someid' could be changed by developers to 'span#someid' and it will break the test.

Reply:
Got it, thank you